iceberg/iceberg-catalog/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -473,11 +512,10 @@ private void cleanupMetadataAndUnlock(CommitStatus commitStatus, String metadata
     }
   }
 
-  @VisibleForTesting
-  void doUnlock(HiveCommitLock lock) {
+  void doUnlock(HiveLock lock) {
```

Comment:
is that used? Looks like it was replaced with `cleanupMetadataAndUnlock` in `Hive: Refactor commit lock mechanism from HiveTableOperations (#6648)`

Reply:
Yes, it is used in `cleanupMetadataAndUnlock`.

Reply:
ok, what is the use of doUnlock() if we just swallow the exception?

Reply:
I assume it is to make cleanupMetadataAndUnlock easier to read. 
TBH, I can only guess the reason why it shallows the exception: both this and cleanupMetadataAndUnlock are used in finally blocks. I assume the author didn't want to add a new exception handling block inside the finally block.

Reply:
@deniskuzZ , can I resolve that conversation?